src/main/java/com/atlassian/db/replica/api/AuroraConnectionDetails.java

```diff
@@ -0,0 +1,44 @@
+package com.atlassian.db.replica.api;
+
+public final class AuroraConnectionDetails {
```

Comment:
Using an interface can make it more testable 🤔.

Reply:
There is no logic in this class (basic data transfer object) so how interface will make it more testable?

Reply:
Yes. It will not make it more testable now. It may change if the API evolve in the future. It can stay as-is.